File: Iniciante/1010 - Cálculo Simples/JavaScript/1010.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const lines = input.split("\n");
  let amountToPay = 0;
  for (let i = 0; i < 2; i++) {
    let [codePartStr, amountPartStr, partValueStr] = lines[i].split(" ");
    let amountPart = parseInt(amountPartStr);
    let partValue = parseFloat(partValueStr);
    amountToPay += amountPart * partValue;
  }
  console.log(`VALOR A PAGAR: R$ ${amountToPay.toFixed(2)}`);
}

main();

File: Iniciante/1010 - Cálculo Simples/Python/1010.py
def main():
    amount_to_pay = 0
    for _ in range(2):
        code_part_str, amount_part_str, part_value_str = input().split()
        amount_part = int(amount_part_str)
        part_value = float(part_value_str)
        amount_to_pay += amount_part * part_value
    print(f"VALOR A PAGAR: R$ {amount_to_pay:.2f}")


if __name__ == "__main__":
    main()
